### DISCUSSION TITLE
Bad gateway with Netbox Tags

### DISCUSSION THREAD

**Discussion by ghsteo** [netbox-community/netbox#13227] [Help Wanted!]
Hello,

We utilize Netbox for our device inventory at our location. We have some VM hosts with trunked Vlans that get updated regularly. The VM Hosts and Vlans both have tags applied to them in Netbox and then we have a python script we run that goes through and ensures that the vlans are synced on the host. However it seems when we have that tag applied to a certain number of VM host interfaces it throws the bad gateway error below:

pynetbox.core.query.RequestError: The request failed with code 502 Bad Gateway but more specific details were not returned in json. Check the NetBox Logs or  investigate this exception's error attribute.gg

Is there any kind of limitation on the amount of recommended tags being used? Wondering the directory on the Netbox logs file referenced above as well. 

Thanks for any assistance!

**Comment by nomad-cyanide** [comment]
Perhaps this is the problem: https://github.com/netbox-community/netbox/issues/14691
Basically a difference in connection-handling in the gunicorn module, which causes connection resets.

**Reply by candlerb** [reply]
You can confirm whether this is the problem by seeing whether you see "Autorestarting worker after current request" in the gunicorn logs (`journalctl -eu netbox`) at the same time as you see the 502 error.

**Reply by ghsteo** [reply]
Thanks, ill give it a shot.
